tensorflow/python/keras/distribute/distribute_strategy_test.py

```diff
@@ -689,6 +696,24 @@ def test_numpy_with_sample_weights(self, distribution):
       model.fit(inputs, targets, sample_weight=sample_weights, epochs=1,
                 steps_per_epoch=2, verbose=1)
 
+  @combinations.generate(
+      combinations.combine(distribution=strategies_minus_tpu, mode=['graph']))
+  def test_sample_weights(self, distribution):
+    with self.cached_session():
+      with distribution.scope():
+        model = get_sample_weights_model()
+        optimizer = rmsprop.RMSPropOptimizer(learning_rate=0.001)
+        loss = 'mse'
+        model.compile(optimizer, loss)
+      
+      inputs = np.array([[0, 1, 2, 3],[1, 3, 4, 5]], np.float32)
+      targets = np.array([[0, 1, 2, 1],[1, 2, 2, 3]], np.float32)
+      sample_weights = np.array([[1, 1, 1, 0],[1, 0, 0, 0]], np.float32)
+
+      result = model.evaluate(inputs, targets, sample_weight=sample_weights, 
+                              verbose=1)
+      self.assertAllEqual(result[0], 0)            
```

Comment:
This looks good. Can you also test the case when you don't pass sample_weights?

Reply:
How about now?